test-clients/src/main/resource/testSystemFiles/throttles-for-jrs-test.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
I believe `throttles-dev.json` is being used for all the JRS tests

Reply:
Yeah, but if you need a customized throttling value for some operations, you need a separate version for that. Otherwise, the change for 'throttles-dev.json' will be applied to all JRS test. 
I probably need to rename it to be more specific.